packages/cloud-cognitive/src/components/ModifiedTabs/ModifiedTabs.js

```diff
@@ -5,93 +5,107 @@
 
 import { pkg } from '../../settings';
+const blockClass = `${pkg.prefix}--modified-tabs`;
 const componentName = 'ModifiedTabs';
 
 // Default values for props
 const defaults = {
-  tabs: [
-    {
-      id: 'tab-1',
-      label: 'Tab 1',
-      content: <div>Iam am the content of tab 1.</div>,
-      unsavedContent: false,
-    },
-  ],
   newTabLabel: 'New tab',
   newTabContent: 'Your new tab will be here soon',
   onNewTab: undefined,
   onCloseTab: undefined,
 };
 
-export let ModifiedTabs = ({
-  tabs = defaults.tabs,
-  newTabLabel = defaults.newTabLabel,
-  newTabContent = defaults.newTabContent,
-  onNewTab = defaults.onNewTab,
-  onCloseTab = defaults.onCloseTab,
-}) => {
-  const handleNewTab = () => {
-    if (onNewTab) {
-      onNewTab();
-      setTimeout(() => {
-        // set focus to the new tab
-        const tab = tabsRef.current.getTabAt(tabs.length);
-        if (tab & tab.tabAnchor) {
-          tab.tabAnchor.focus();
-        }
-      });
-    }
-  };
+export let ModifiedTabs = forwardRef(
+  (
+    {
+      className,
+      tabs = defaults.tabs,
+      newTabLabel = defaults.newTabLabel,
+      newTabContent = defaults.newTabContent,
+      onNewTab = defaults.onNewTab,
+      onCloseTab = defaults.onCloseTab,
+    },
+    ref
+  ) => {
+    const tabsRef = useRef();
+    const localRef = ref || tabsRef;
 
-  const handleClose = (id) => {
-    if (onCloseTab) {
-      onCloseTab(id);
-    }
-  };
+    const handleNewTab = () => {
+      if (onNewTab) {
+        onNewTab();
+        setTimeout(() => {
+          // set focus to the new tab
+          const addNewTabButton = localRef.current.querySelector(
+            `.${blockClass}__tab-new`
+          );
+          const tab = addNewTabButton?.previousSibling;
+          tab?.focus();
+        });
+      }
+    };
 
-  const tabsRef = useRef(null);
+    const handleClose = (id) => {
+      if (onCloseTab) {
+        onCloseTab(id);
+      }
+    };
 
-  return (
-    <Tabs className="modified-tabs" ref={tabsRef}>
-      {tabs.map((tab) => (
-        <Tab
-          id={tab.id}
-          key={tab.id}
-          label={
-            <ModifiedTabLabelWithClose
-              label={tab.label}
-              onClose={() => handleClose(tab.id)}
-              unsavedContent={tab.unsavedContent}
-            />
-          }
-        >
-          <div className="some-content">{tab.content}</div>
-        </Tab>
-      ))}
-      <Tab
-        id="modified-tabs__tab-new"
-        label={<ModifiedTabLabelNew label={newTabLabel} />}
-        onClick={handleNewTab}
-        onKeyUp={(ev) => ev.keyCode === 32 && handleNewTab()}
-        role="button"
-      >
-        <div className="some-content">{newTabContent}</div>
-      </Tab>
-    </Tabs>
-  );
-};
+    return (
+      <div className={cx(blockClass, className)} ref={localRef}>
+        <Tabs className="modified-tabs">
+          <TabList aria-label="Modified tab list temp aria label">
```

Comment:
Was this left in by mistake? Expecting we want an actual aria-label here?

Reply:
Yes, left by mistake. I'll update this so it's passed as a prop